<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP Mission Creator (c) PMC</string>
  </property>
  <widget class="QSpinBox" name="numberOfPlayers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Players</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>74</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>COOP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Team</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scont</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CTI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CTF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unknown</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="missionName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PMC Mission v1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="missionDescription">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Infantry only, random enemies.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="terrain">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Utes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chernarus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Takistan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zargabad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Desert_E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shapur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ProvingGrounds</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMC_51km_Desert</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMC_Rattler</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Terrain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createMission">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QComboBox" name="respawn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>74</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BIRD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INSTANT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BASE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GROUP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SIDE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Respawn</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="respawnDelay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>900</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Respawn delay</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="disableAI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>disableAI</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showGPS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>showGPS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>onLoadIntro</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="onLoadIntro">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Ready!</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="onLoadIntroTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>onLoadIntroTime</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="onLoadMissionTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>onLoadMissionTime</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="onLoadMission">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mission by PMC!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>onLoadMission</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
